src/main/java/fact/io/hdureader/FITSStream.java

```diff
@@ -47,10 +47,21 @@ public void init() throws Exception {
         //get calibration constants they are stored in the first (and only) row of this hdu.
         HDU offsetHDU = fits.getHDU("ZDrsCellOffsets");
         if (offsetHDU != null){
-            ZFITSHeapReader reader = ZFITSHeapReader.forTable(offsetHDU.getBinTable());
-            boolean ignoreWrongTileHeader;
-            OptionalTypesMap<String, Serializable> row = reader.getNextRow(ignoreWrongTileHeader=true);
+            Boolean ztable = offsetHDU.header.getBoolean("ZTABLE").orElse(false);
```

Comment:
is this for a case when the zdrscelloffsets are not compressed? does that ever happen?

Reply:
yes in the same data as the one with ZTILELEN=100